MVEE/Src/MVEE_syscalls_handlers.cpp

```diff
@@ -6407,7 +6415,7 @@ PRECALL(poll)
     CHECKPOINTER(1);
     CHECKARG(2);
     CHECKARG(3);
-    CHECKBUFFER(1, sizeof(struct pollfd) * ARG2(0));
+	CHECKPOLLFD(1, ARG2(0));
```

Comment:
The indentation here got messed up, do you want tabs or spaces @stijn-volckaert ?